msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -154,12 +154,12 @@
     <None Update="Platforms\net45\WindowsFormsWebAuthenticationDialogBase.cs" SubType="Form" />
     <None Update="Platforms\net45\SilentWindowsFormsAuthenticationDialog.cs" SubType="Form" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid7'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid8.1'">
```

Comment:
><ItemGroup [](start = 2, length = 10)

minor: content of this item group could be moved up to the android item group above